packages/zen-bull-netting/src/ZenBullNetting.sol

```diff
@@ -140,4 +171,35 @@ contract ZenBullNetting is Ownable, EIP712 {
 
         otcPriceTolerance = _otcPriceTolerance;
     }
+
+    /**
+     * @notice queue USDC for deposit into crab strategy
+     * @param _amount USDC amount to deposit
+     */
+    function queueWeth(uint256 _amount) external {
```

Comment:
I think we probably want users to queue ETH generally (with maybe the advanced option to queue ETH)

I think could create a wrapper function that just wraps and then calls queueWeth.

Right now deposits and withdrawals are in ETH not WETH for zen, so should be consistent.

If you want can be in another PR, but lets add a linear issue/ticket for it and add to spec.